portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/applicationdisplaytemplate/ApplicationDisplayTemplatesWithStaging.testcase

```diff
@@ -54,6 +54,59 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-130051. Widget templates can be used to modify the look of a specific application."
+	@priority = "4"
+	test CustomWidgetTemplateWithLocalLiveStaging {
+		task ("Add a widget templates in staging site") {
+			ApplicationDisplayTemplates.openWidgetTemplatesAdmin(siteURLKey = "test-site-name-staging");
+
+			var templateScript = '''<a>${content.getData()}</a>''';
+
+			ApplicationDisplayTemplates.addCP(
+				adtType = "Navigation Menu",
+				templateScript = "${templateScript}");
+		}
+
+		task ("Add a content page based on Blank") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name (Staging)",
+				layoutName = "Test Page Name",
+				site = "false",
+				type = "content");
+		}
+
+		task ("Add a Navigation Menu widget to content page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name Staging");
+
+			PageEditor.addWidget(portletName = "Navigation Menu");
+		}
+
+		task ("Configure the Navigation Menu widget to use the Navigation Menu Template") {
+			NavigationMenusWidget.gotoNavigationMenuWidgetConfig();
+
+			NavigationMenusWidget.selectDisplayTemplate(templateName = "Navigation Menu Test ADT");
```

Comment:
Use other OOTB display template, for example, List Menu, Tabs
![image](https://user-images.githubusercontent.com/52661397/125178109-3ae15000-e214-11eb-907b-9fadcbd128eb.png)